### PR TITLE
feat: add streams button

### PR DESCRIPTION
**Pull request by LeviSnoot**
I'm not sure where I messed up but it won't add `rel=me` to any links using this setup, including Mastodon and Firefish, so I need help solving that before we can merge.

### CODE REVIEW

**Comment by LeviSnoot** [comment]
Once correctly implemented, this should fix #600

File: resources/views/littlelink.blade.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 {{-- Fediverse rel="me" links --}}
 @php
-  $relMe = "mastodon, firefish";
+  $relMe = "mastodon, firefish, streams";
```
Comment (LeviSnoot) [inline resources/views/littlelink.blade.php:10]: Issue is definitely related to this and possibly the next line. I'm just not sure what the problem is.

**Comment by JulianPrieber** [comment]
This adds the rel me thing to the HTML head
https://github.com/LinkStackOrg/LinkStack/blob/b3533d82cd40bee3eb3a33a33ff8a0422aba630b/resources/views/littlelink.blade.php#L5

**Comment by JulianPrieber** [comment]
Is this it?
https://codeberg.org/streams/streams

**Comment by LeviSnoot** [comment]
> This adds the rel me thing to the HTML head
> 
> https://github.com/LinkStackOrg/LinkStack/blob/b3533d82cd40bee3eb3a33a33ff8a0422aba630b/resources/views/littlelink.blade.php#L5

I'm not sure how it manages to work in the current version then since there's no other reference to either mastodon or firefish in that file. 

> Is this it? https://codeberg.org/streams/streams

Yup, that's the one.

**Comment by JulianPrieber** [comment]
If you add a button name to the array this loop is gonna look through every link that would be displayed on the page:
```
@foreach($links as $link)
  @if(in_array($link->name, $relMeList))
    <link href="{{$link->link}}" rel="me">
  @endif
@endforeach
```
If a link matches one in the list, this is added inside the head tag of the page:
`<link href="[the link]" rel="me">`

This will not be added to the links to the sites just a separate independent tag inside the head that can only be seen in the rendered HTML.

This will then look something like this:
BVmXiBJ8uL

Something like Mastodon picks up on this and confirms the page as verified.

Doing it this way is easier than have a separate check in every button that needs this.

**Comment by LeviSnoot** [comment]
Oh okay that makes perfect sense, I'll check if the current draft functions as intended in a bit and change the status of the PR if it does.

**Comment by LeviSnoot** [comment]
image

Checked now and yeah it's all there :100:

**Pull request merged**